frontend/src/Event/MeetEventDisplay.jsx

```diff
@@ -44,6 +44,9 @@ const MeetEventDisplay = () => {
   const [selectedEventIndex, setSelectedEventIndex] = useState(null);
   const [navegationDirection, setNavegationDirection] = useState(null);
 
+  //GenerateHeats states
+  const [generateHeatTrigger, setGenerateHeatTrigger] = useState(0);
```

Comment:
I think this state variable would be better defined as a boolean with a more explicit name like areHeatsJustCreated, isGenerateHeatsProcessCompleted, something like that, what do you think?

Reply:
This state is used to re-fetch the events data every time we generate heats. 
In this case, a number like generateHeatTrigger is more effective, as it can trigger useEffect every time the state changes, without needing to reset a boolean back and forth.
I could change the name, maybe to something like dataRefreshAfterGenerateHeats? What do you think?